var/spack/repos/builtin/packages/superlu/package.py

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Superlu(Package):
+    """SuperLU is a general purpose library for the direct solution of large,
+    sparse, nonsymmetric systems of linear equations on high performance
+    machines. SuperLU is designed for sequential machines."""
+
+    homepage = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/#superlu"
+    url      = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_5.2.1.tar.gz"
+
+    version('5.2.1', '3a1a9bff20cb06b7d97c46d337504447')
+
+    depends_on('blas')
+
+    def install(self, spec, prefix):
+        cmake_args = [
+            '-DCMAKE_C_FLAGS=-fPIC',
+            '-DCMAKE_Fortran_FLAGS=-fPIC',
```

Comment:
You can use `-DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON` to get the proper flag for any compiler.